sdk/storage/tests/data_lake.rs

```diff
@@ -14,29 +17,61 @@ async fn test_data_lake_file_system_functions() -> Result<(), Box<dyn Error + Se
     let master_key = std::env::var("ADSL_STORAGE_MASTER_KEY")
         .expect("Set env variable ADSL_STORAGE_MASTER_KEY first!");
 
-    let file_system_name = "azuresdke2etest";
+    let now = Utc::now();
+    let file_system_name = format!("azurerustsdk-e2etest-datalake-{}", now.timestamp());
 
     let http_client = new_http_client();
 
     let storage_account_client =
         StorageAccountClient::new_access_key(http_client.clone(), &account, &master_key);
 
+    let resource_id = "https://storage.azure.com/";
+    println!("getting bearer token for '{}'...", resource_id);
+    let bearer_token = DefaultCredential::default().get_token(resource_id).await?;
+
+    println!("token value = {}", bearer_token.token.secret());
+    println!("token expires on= {}", bearer_token.expires_on);
+    println!();
+
     let data_lake = storage_account_client
         .as_storage_client()
-        .as_data_lake_client(account)?;
+        .as_data_lake_client(account, bearer_token.token.secret().to_owned())?;
 
-    let file_system = data_lake.as_file_system_client(file_system_name)?;
+    let file_system = data_lake.as_file_system_client(&file_system_name)?;
 
     let mut properties = Properties::new();
     properties.insert("AddedVia", "Azure SDK for Rust");
     properties.insert("CreatedAt", chrono::Utc::now().to_string());
-    let response = file_system
+    println!("creating file system '{}'...", &file_system_name);
+    let create_file_system_response = file_system
         .create()
         .properties(&properties)
         .execute()
         .await?;
-    println!("repsonse == {:?}", response);
+    println!(
+        "create file system response == {:?}",
+        create_file_system_response
+    );
+    println!();
```

Comment:
Ah, yes. I had converted these from assert to println so that there is no debris left behind when an earlier assertion fails. But I since learned that the e2e tests don't run in CI, so that's no longer a concern and I'll convert it back